nixos/modules/services/mail/maddy.nix

```diff
@@ -391,6 +411,8 @@ in {
           ) cfg.tls.certificates)} ${optionalString (cfg.tls.extraConfig != "") ''
             { ${cfg.tls.extraConfig} }
           ''}
+        '' else if (cfg.tls.loader == "acme") then ''
+          tls acme { ${cfg.tls.extraConfig} }
```

Comment:
I am not sure if this is the correct syntax? Looking at https://maddy.email/reference/tls-acme/#dns-providers, the `tls.loader.acme local_tls` examples does look a bit different (and I can't see any reference to the `local_tls` or any usage of the tls introduced by acme in here)